content/help/bugs/index.md

```diff
@@ -9,8 +9,50 @@ aliases:
 
 Oh no! You found a bug?
 
-Search our [issue database](https://github.com/istio/istio/issues/) to see if we already know about
-your problem and learn about when
-we think we can fix it. If you don't find your problem in the database, please open a [new issue](https://github.com/istio/istio/issues/new/choose)
-and let us know what's
-going on.
+Search our [issue database](https://github.com/istio/istio/issues/) to see if
+we already know about your problem and learn about when we think we can fix
+it. If you don't find your problem in the database, please open a [new
+issue](https://github.com/istio/istio/issues/new/choose) and let us know
+what's going on.
+
+If you're running on Kubernetes, consider including a [cluster state
+archive](#generating-a-cluster-state-archive) in your bug report.
+
+## Generating a cluster state archive
+
+For convenience, you can run a dump script to produce an archive containing
+all of the needed state from your Kubernetes cluster:
+
+* Run via `curl`:
+
+    {{< text bash >}}
+    $ curl https://raw.githubusercontent.com/istio/istio/release-1.0/tools/dump_kubernetes.sh | sh -s -- -z
```

Comment:
replace release-1.0 with {{< branch_name >}}. This will insert the appropriate branch name for the current release.

Reply:
Oh nice. Done.